### PR TITLE
Add Chef Infra Client 15.8 release notes

### PR DESCRIPTION
**Pull request by tas50**
Release notes for 15.8

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
This is WIP since there's a few more things that will probably get merged before next week when we ship.

**Comment by tas50** [comment]
We're probably feature complete at this point.

**Comment by tas50** [comment]
I was afraid to add that comma because I always over comma the world

**Comment by IanMadd** [comment]
> I was afraid to add that comma because I always over comma the world

Commas are finicky.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,8 +1,127 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
-# UNRELEASED
+# Chef Infra Client 15.8
 
-### sysctl now accepts a comments parameter
+## New notify_group functionality
+
+Chef Infra Client now includes a new `notify_group` feature that can be used to extract multiple common notifies out of individual resources to reduce duplicate code in your cookbooks and custom resources. Previously cookbook authors would often use a `log` resource to achieve a similar outcome, but using the log resource results in unnecessary Chef Infra Client log output. The `notify_group` method produces no additional logging, but fires all defined notification when the `:run` action is set.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
Chef Infra Client now includes a new `notify_group` feature that can be used to extract multiple common notifies out of individual resources to reduce duplicate code in your cookbooks and custom resources. Previously cookbook authors would often use a `log` resource to achieve a similar outcome, but using the log resource results in unnecessary Chef Infra Client log output. The `notify_group` method produces no additional logging, but fires all defined notifications when the `:run` action is set.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,8 +1,127 @@
...
+    variables node["crude"]
+    notifies :run, "notify_group[crude_stop_and_start]", :immediately
+  end
+```
+
+## Chef InSpec 4.18.85
+
+Chef InSpec has been updated from 4.18.39 to 4.18.85. This release includes a large number of bug fixes in addition to some great resource enhancements:
+
+* The service resource features new support for yocto-based linux distributions. Thank you to @michaellihs 1 for this addition!
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
* The service resource features new support for yocto-based linux distributions. Thank you to [@michaellihs](https://github.com/michaellihs) for this addition!
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,8 +1,127 @@
...
+
+## New Cookbook Helpers
+
+New helpers have been added to make writing cookbooks easier.
+
+### Platform Version Helpers
+
+New helpers for checking platform versions have been added. These helpers return parsed version strings so there's no need to convert the returned values to Integers or Floats before comparing them. Additionally, comparisons with version objects properly understand the order of versions so `5.11` will compare as larger than `5.9`, whereas converting those values to Floats would result in `5.9` being larger than `5.11`.
+
+* `windows_nt_version` returns the NT kernel version, which often differs from Microsoft's marketing versions. This helper offers a good way to find desktop and server releases that are based on the same codebase. For example NT 6.3 is both Windows 8.1 and Windows 2012 R2.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
* `windows_nt_version` returns the NT kernel version which often differs from Microsoft's marketing versions. This helper offers a good way to find desktop and server releases that are based on the same codebase. For example, NT 6.3 is both Windows 8.1 and Windows 2012 R2.
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,8 +1,127 @@
...
+* `powershell_version` returns the version of PowerShell installed on the system.
+* `platform_version` returns the value of node['platform_version'],
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
* `platform_version` returns the value of node['platform_version'].
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,8 +1,127 @@
...
+* `openvz?` - if the node is an openvz guest
+* `openvz_host?` - if the node is an openvz host
+* `guest?` - if the node is detected as any kind of guest
+* `hypervisor?` - if the node is detected as being any kind of hypervisor
+* `physical?` - the node is not running as a guest (may be a hypervisor or may be bare-metal)
+* `vagrant?` - attempts to identify the node as a vagrant guest (this check may be error-prone)
+
+### include_recipe? helper
+
+chef-sugar's `include_recipe?` has been added to Chef Infra Client, providing a simple way to see if a recipe has been included on a node already.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
chef-sugar's `include_recipe?` has been added to Chef Infra Client providing a simple way to see if a recipe has been included on a node already.
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,8 +1,127 @@
...
+
+## Updated Resources
+
+### ifconfig
+
+The `ifconfig` resource now supports the newer `ifconfig` release that ships in Debian 10.
+
+### mac_user
+
+The `mac_user` resource used when creating a user on Mac systems has been improved to work better with macOS Catalina (10.15). The resource now properly looks up the numeric GID when creating a user, once again supports the `system` property, and includes a new `hidden` property which prevents the user from showing on the login screen. Thanks [@chilcote](https://github.com/chilcote) for these fixes and improvements.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
The `mac_user` resource, used when creating a user on Mac systems, has been improved to work better with macOS Catalina (10.15). The resource now properly looks up the numeric GID when creating a user, once again supports the `system` property, and includes a new `hidden` property which prevents the user from showing on the login screen. Thanks [@chilcote](https://github.com/chilcote) for these fixes and improvements.
```

**Pull request merged**